components/fileselect/templates.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Templates
+page_title: FileSelect - Templates
+description: Templates in the FileSelect for Blazor.
+slug: fileselect-templates
+tags: telerik,blazor,fileselect,templates
+published: True
+position: 30
+---
+
+# FileSelect Templates
+
+The `SelectFilesButtonTemplate` allows you to modify the "Select Files..." button. It lets you change the default text of the button and include a custom content like an [icon](https://docs.telerik.com/blazor-ui/common-features/icons) or image.
+
+>caption Using FileSelect SelectFilesButtonTemplate
+
+```CSHTML
+<div>
+    <TelerikFileSelect AllowedExtensions="@AllowedExtensions">
+        <SelectFilesButtonTemplate>
+            <TelerikFontIcon Icon="@FontIcon.Upload" />
+            MyText
```

Comment:
Avoid so obvious "hello world" approach in our examples. Try something like "Click to Select Files for Upload" or anything you prefer.